src/expandable-section/styles.scss

```diff
@@ -180,3 +181,12 @@
     @include styles.form-focus-element(awsui.$border-radius-control-default-focus-ring);
   }
 }
+
+.click-target {
+  cursor: pointer;
+}
+
+.description-default {
+  // Indent the description by exactly the same amount as the header, which is pushed to the right by the icon.
```

Comment:
Visual design review surfaced that all descriptions in expandable section variants should be indented. Container variant will be handled in a follow-up PR.